types/src/shared/rate_limiter.ts

```diff
@@ -3,37 +3,49 @@ import { v4 as uuidv4 } from "uuid";
 import { MaxMessagesTimeframeType } from "../front/plan";
 import { LoggerInterface } from "../shared/logger";
 import { redisClient } from "../shared/redis_client";
+import { Err, Ok, Result } from "./result";
 import { getStatsDClient } from "./statsd";
 
 export class RateLimitError extends Error {}
 
+async function getRedisClient(redisUri?: string) {
```

Comment:
IIRC we already have such function right, could we reuse it ? Or move the existing one to `types`?

Reply:
Yep follow up PR to extract redis from types.

Reply:
🙏🏼